cmd/query/main.go

```diff
@@ -65,6 +66,18 @@ func main() {
 
 			metricsFactory := xkit.Wrap("jaeger-query", expvar.NewFactory(10))
 
+			tracer, closer, err := jaegerClientConfig.Configuration{
+				Sampler: &jaegerClientConfig.SamplerConfig{
+					Type:  "probabilistic",
```

Comment:
use `const` instead of probabilistic, it's cheaper.

Reply:
actually, I think this needs to be overridable vie cmd line switches

Reply:
This will be via https://github.com/uber/jaeger-client-go/issues/206 no? I think we should have one set of variables for all the Jaeger bindings, so that its really easy to configure jaeger enabled components in one consistent fashion. That hasn't been done yet though.

I can switch this to const easily - I think its worth noting though that this is the code thats in the all-in-one entry point. Should I change that at the same time?

Given that this is a very simple and relatively low-use UI: scales O(developers) not O(users), I think a const of 1 is fine to start with.